src/test/java/com/lpvs/service/GitHubServiceTest.java

```diff
@@ -0,0 +1,376 @@
+            webhookConfig.setRepositoryName("LPVS");
+            webhookConfig.setRepositoryOrganization("Samsung");
+
+            try {
+                when(mocked_instance_gh.getRepository(webhookConfig.getRepositoryOrganization() + "/" + webhookConfig.getRepositoryName())).thenReturn(mocked_repo);
+            } catch (IOException e) {
+                LOG.error("mocked_repo.getRepository error " + e);
+            }
+            try {
+                when(mocked_repo.getLicense()).thenReturn(mocked_license);
+            } catch (IOException e) {
+                LOG.error("mocked_repo.getLicense error " + e);
+            }
+            when(mocked_license.getKey()).thenReturn(test_license_key);
+        }
+
+        @Test
+        public void testGetRepositoryLicense__ApiUrlPresentLisencePresent() {
+
+            try (MockedStatic<GitHub> mocked_static_gh = mockStatic(GitHub.class)) {
+                mocked_static_gh.when(() -> GitHub.connectToEnterpriseWithOAuth(GH_API_URL, GH_LOGIN, GH_AUTH_TOKEN)).thenReturn(mocked_instance_gh);
+
+                // main test
+                assertEquals(test_license_key, gh_service.getRepositoryLicense(webhookConfig));
+
+                // verification of calling methods on `Mock`s
+                // `mocked_static_gh` verify
+                mocked_static_gh.verify(() -> GitHub.connectToEnterpriseWithOAuth(GH_API_URL, GH_LOGIN, GH_AUTH_TOKEN), times(1));
+                mocked_static_gh.verifyNoMoreInteractions();
+
+                // `mocked_instance_gh` verify
+                try {
+                    verify(mocked_instance_gh, times(1)).getRepository(webhookConfig.getRepositoryOrganization() + "/" + webhookConfig.getRepositoryName());
+                } catch (IOException e) {
+                    LOG.error("mocked_instance_gh.getRepository error " + e);
+                }
+                verifyNoMoreInteractions(mocked_instance_gh);
+
+                // `mocked_repo` verify
+                try {
+                    verify(mocked_repo, times(1)).getLicense();
+                } catch (IOException e) {
+                    LOG.error("mocked_repo.getLicense error " + e);
+                }
+                verifyNoMoreInteractions(mocked_repo);
+
+                // `mocked_license` verify
+                verify(mocked_license, times(1)).getKey();
+                verifyNoMoreInteractions(mocked_license);
+            }
+        }
+    }
+
+
+    @Nested
+    class TestGetRepositoryLicense__ApiUrlPresentLisenceAbsent {
+
+        String GH_LOGIN = "test_login";
+        String GH_AUTH_TOKEN = "test_auth_token";
+        String GH_API_URL = "test_api_url";
+        GitHubService gh_service = new GitHubService(GH_LOGIN, GH_AUTH_TOKEN, GH_API_URL);
+        WebhookConfig webhookConfig;
+        GitHub mocked_instance_gh = mock(GitHub.class);
+        GHRepository mocked_repo = mock(GHRepository.class);
+        // GHLicense mocked_license = mock(GHLicense.class);
```

Comment:
Why do you leave a commented-out code?

Reply:
Commented code was left intentionally. To show the difference between test cases. Intention was to show difference between `LicensePresent` and `LicenseAbsent` test cases in this case. 

Should I delete it?

Reply:
Resolved. I will wait for the discussion below, then approve it. 😄

Reply:
I agree, please keep this commented code as is.